draft-irtf-cfrg-frost.md

```diff
@@ -695,6 +695,27 @@ The value of the contextString parameter is "FROST-RISTRETTO255-SHA512".
     the output to a Scalar as described in {{!RISTRETTO, Section 4.4}}.
   - H3(m): Implemented by computing H(contextString \|\| "digest" \|\| m).
 
+## FROST(edwards448, SHAKE256)
```

Comment:
This is `edwards448` here and `ed448` below, should we pick one?

Reply:
Heh, yeah. Do you have a preference? 🤷